requirements/base.in

```diff
@@ -4,3 +4,5 @@
 Django
 XBlock
 xblock-utils
+# TODO: Change this once codejail is published to pypi
+-e git+https://github.com/edx/codejail.git@3.0.0#egg=codejail
```

Comment:
Will pinning this here cause conflicts with the version specified in `edx-platform`?

I _think_ based on my reading of `setup.py:is_requirement` [1] that it should be fine;
this line _should_ be ignored, for purposes of the package's `install_requires` [2], since it starts with `-e`, right?

Just want to check...

- [1] https://github.com/edx/xblock-sql-grader/blob/master/setup.py#L44
- [2] https://github.com/edx/xblock-sql-grader/blob/master/setup.py#L54

Reply:
Right, it's a workaround until codejail gets pushed to pypi. If you install the block, it will skip installing codejail. But since edx-platform requires codejail, everything will work as expected.
If you're developing on the block or running tests, you'd pip install the requirements, which will install codejail from git.